### ISSUE TITLE
Unable to add IPv6 Service_Address using a defaultRouteDomain other than 0

### ISSUE THREAD

**Issue by manhac** [F5Networks/f5-appsvcs-extension#324]
### Environment
 * Application Services Version: 3.19.1 and 3.21.0
 * BIG-IP Version: 15.1.0.4

### Summary
My goal is to create forwarding virtual servers that pass all IPv4 and IPv6 traffic (as described in "Emulating stateless IP routing with BIG-IP LTM forwarding virtual servers" https://support.f5.com/csp/article/K7595).
As I need to control the ICMP response and ARP behaviour, I have to specify and reference a Service_Address object in the Service_Forwarding object.

Generally this is possible using IPv4 (0.0.0.0/0) or IPv6 (::/0) as the virtualAddress. However, if I change the defaultRouteDomain from 0 to another value, then it stops working for IPv6, but it still works for IPv4. If "::/0" is directly specified within virtualAddresses in the Service_Forwarding object (without a Service_Address object) the API also accepts this request -- but then I do not have control over the ICMP response...

### Steps To Reproduce
1. Create a route domain (in this example: 100).

2. Submit the following declaration:
```json
{
  "schemaVersion": "3.0.0",
  "class": "ADC",
  "forwardingvirtualserver": {
    "class": "Tenant",
    "defaultRouteDomain": 100,
    "forwardingvirtualserver": {
      "class": "Application",
      "template": "generic",
      "virtual_forwarding_address": {
        "class": "Service_Address",
        "virtualAddress": "::/0"
      },
      "forward_any_to_any": {
        "class": "Service_Forwarding",
        "forwardingType": "ip",
        "virtualAddresses": [
          {
            "use": "virtual_forwarding_address"
          }
        ],
        "virtualPort": 0,
        "layer4": "any"
      }
    }
  }
}
```

2. Observe the following error response:
```json
{
    "results": "HTML Tag-like Content in the Request URL/Body",
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.0.0",
        "id": "autogen_64ddd8b2-0e59-4951-bd91-1c456eec1aa6",
        "updateMode": "selective",
        "controls": {
            "archiveTimestamp": "2020-08-17T09:50:29.985Z",
            "class": "Controls",
            "logLevel": "error",
            "trace": false,
            "traceResponse": false
        }
    },
    "code": 422
}
```

3. Observe the following error in **/var/log/restnoded/restnoded.log**:
Mon, 17 Aug 2020 10:52:30 GMT - severe: [appsvcs] {"message":"Declaration failed: A port number or service name is missing for \\\"::%100:0\\\". Please specify a port number or service name using the syntax \\\"::%100:0.<port>\\\".","level":"error"}

**Comment by dstokesf5** [comment]
Thank you for reporting this issue. I have confirmed it and added it to our internal product backlog as AUTOTOOL-1837.

**Comment by jovnh** [comment]
Hello, any updates on this? We are facing the same issue

**Comment by dstokesf5** [comment]
It looks like AUTOTOOL-1837 is still in our backlog. I will try to increasing the priority of it.

**Comment by dstokesf5** [comment]
This issue has been resolved by the 3.29.0 release.
